backend/app/agents/devrel/tools/search_tool/ddg.py

```
@@ -7,37 +7,94 @@
+                continue
             
-            results = []
-            for result in response or []:
-                results.append({
-                    "title": result.get("title", ""),
-                    "content": result.get("body", ""),
-                    "url": result.get("href", ""),
-                    "score": 0
-                })
-            return results
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)